src/Microsoft.Extensions.Hosting/baseline.netcore.json

```diff
@@ -0,0 +1,397 @@
+{
+  "AssemblyIdentity": "Microsoft.Extensions.Hosting, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
Isn't the baseline supposed to be the prior public release? This component is new, it hasn't shipped before.

Reply:
Is that true of the other component too?

Reply:
No, WindowsServices 2.0 had a net461 version, it should have a baseline. Only the netcoreapp2.0 TFM is new.

Reply:
Do we have a way to differentiate the baselines based on the TFM?

Reply:
We differentiate Core vs Framework based on the name of the baseline file generated.